tests/test_aligner.yml

```diff
@@ -48,40 +81,85 @@
     - preprocessing
   files:
     - path: results/csv/markduplicates.csv
+      md5sum: 0d6120bb99e92f6810343270711ca53e
     - path: results/csv/markduplicates_no_table.csv
+      md5sum: 2a2d3d4842befd4def39156463859ee3
     - path: results/csv/recalibrated.csv
+      md5sum: 42628ec994c16f565e5407b40a9c1ac3
     - path: results/multiqc
     - path: results/preprocessing/markduplicates/test/test.md.cram
+    # binary changing on reruns
     - path: results/preprocessing/markduplicates/test/test.md.cram.crai
+    # binary changing on reruns
     - path: results/preprocessing/recal_table/test/test.recal.table
+      md5sum: 75ba4376a17ca69c5134153302f82e92
     - path: results/preprocessing/recalibrated/test/test.recal.cram
+    # binary changing on reruns
     - path: results/preprocessing/recalibrated/test/test.recal.cram.crai
+    # binary changing on reruns
     - path: results/reference/dbsnp/dbsnp_146.hg38.vcf.gz.tbi
+      md5sum: 628232d0c870f2dbf73c3e81aff7b4b4
     - path: results/reference/dict/genome.dict
+      md5sum: 2433fe2ba31257337bf4c4bd4cb8da15
     - path: results/reference/dragmap/hash_table.cfg
+      # hash_table.cfg contains many strings which we could test for - which do we want to test?
+      contains:
+        [
+          "reference_sequences  = 1",
+          "reference_len        = 368640",
+          "reference_len_raw    = 40001",
+          "reference_len_not_n  = 40001",
+          "reference_alt_seed   = 204800",
+        ]
     - path: results/reference/dragmap/hash_table.cfg.bin
+    # binary changing on reruns
     - path: results/reference/dragmap/hash_table.cmp
+      md5sum: 1caab4ffc89f81ace615a2e813295cf4
     - path: results/reference/dragmap/hash_table_stats.txt
+      # hash_table_stats.txt contains many string which we could test for - which do we want to test?
```

Comment:
```suggestion
      # hash_table_stats.txt  test for strings
```

Reply:
The file `hash_table_stats.txt` contains whole bunch of values which could be tested, but too many for us to test them all. Are there any particular values which you would recommend that we test?

[hash_table_stats.txt](https://github.com/nf-core/sarek/files/9357873/hash_table_stats.txt)